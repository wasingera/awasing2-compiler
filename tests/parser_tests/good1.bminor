x: integer = (y ^ 5 + 4) ^ (6 + 3 / 2 *5);

f: function integer () = {
if (x >= 5) {
    for (;x > 10; x++)
        if (true)
            print x, y;
}
}
