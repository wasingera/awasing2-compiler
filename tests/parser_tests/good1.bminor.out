parse error: syntax error
Parse failed: if
